force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -212,10 +254,11 @@ public with sharing class ClassesObjectsMethods {
      * @return The a positive number years between the two dates, or null if either date is null.
      */
     public static Integer calculateYearDifference(Date d1, Date d2) {
-
-        // Subtract the difference in years
-
-        return null; // Replace null with the variable you used to store the result
+        if (d1 == null || d2 == null) {
+            return null;
+        } else {
+            return (d2.year() - d1.year()); 
```

Comment:
The method instructions ask for a positive number. How can you use `Math.abs()` in this line to fix that?

Reply:
Yup, missed that one